features/geometrisch-zoeken.feature

```diff
@@ -0,0 +1,327 @@
+    het maakt niet uit of geometrie A of geometrie B de resource geometrie is.
+    Als intersects true oplevert, dan wordt de resource waar de geometrie onderdeel van is, geretourneerd.
+
+    Referenties:
+    - Haal Centraal Common - foutafhandeling.feature
+    
+    Het geometrisch zoeken met een bounding box geometrie kan met de parameter: bbox.
+    
+    Het geometrisch zoeken met een polygoon geometrie kan met de parameter: polygoon.
+
+    Het geometrisch zoeken met een punt geometrie kan met de parameter: locatie.
+    Het is echter ook mogelijk om een bounding box op te geven waarbij de coördinaten van de linker onderhoek en de rechter bovenhoek exact gelijk zijn.
+
+    In de hierna volgende voorbeelden is een vereenvoudigde representatie gebruikt voor coördinaten.
+    
+    ![Zoeken met locatie voorbeelden](../images/locatie-voorbeelden.png)    
+    ![Zoeken met bounding box voorbeelden](../images/bbox-voorbeelden.png)    
+    ![Zoeken met polygoon voorbeelden](../images/locatie-voorbeelden.png)      
+
+  Achtergrond:
+
+    Gegeven de volgende verzameling van objecten
+
+    | identificatie | geometrie                          |
+    |---------------|------------------------------------|
+    | 0001          | (0,0),   (10,0),  (10,10), (0,10)  |
+    | 0002          | (12,0),  (22,0),  (22,10), (12,10) |
+    | 0003          | (12,12), (22,12), (22,22), (12,22) |
+    | 0004          | (0,8),   (10,8),  (10,18), (0,18)  |
+    
+  Scenario: GET met locatie, Content-Crs (optioneel) en Accept-Crs (optioneel) één resource
+  
+    Als er met de volgende parameters wordt gezocht:
+    | locatie | Content-Crs | Accept-Crs |
+    |---------|-------------|------------|
+    | (5,5)   | -           | -          |
+    | (5,5)   | EPSG:28992  | -          |
+    | (5,5)   | -           | EPSG:28992 |
+    | (5,5)   | EPSG:28992  | EPSG:28992 |
+    | (10,5)  | -           | -          |
+    | (10,5)  | EPSG:28992  | -          |
+    | (10,5)  | -           | EPSG:28992 |
+    | (10,5)  | EPSG:28992  | EPSG:28992 |
+    En de provider ondersteund het CRS in Content-Crs
+    En de provicer ondersteund het CRS in Accept-Crs
+    En de resource geometrie intersects met de opgegeven bounding box geometrie
+    Dan bevat het resultaat de volgende objecten:
+    | identificatie | geometrie                      |
+    |---------------|--------------------------------| 
+    | 0001          | (0,0), (10,0), (10,10), (0,10) |
+
+    Bijvoorbeeld:
+      /panden?locatie=5,5
+
+  Scenario: GET met locatie, Content-Crs (optioneel) en Accept-Crs (optioneel) meerdere resources
+  
+    Als er met de volgende parameters wordt gezocht:
+    | locatie | Content-Crs | Accept-Crs |
+    |---------|-------------|------------|
+    | (5,9)   | -           | -          |
+    | (5,9)   | EPSG:28992  | -          |
+    | (5,9)   | -           | EPSG:28992 |
+    | (5,9)   | EPSG:28992  | EPSG:28992 |
+    En de provider ondersteund het CRS in Content-Crs
```

Comment:
we moeten hier toch juist laten zien dat de provider dit wel of niet ondersteunt? Dat is toch wat we moeten beschrijven en wat uit de uitkomst van het scenario moet blijken?